SandBox/script.js

```diff
@@ -0,0 +1,291 @@
+/**
+ * Сортировка объектов
+ *
+ * Напишите код, который отсортирует массив объектов people по полю age.
+ * Выведите список имён в массиве после сортировки.
+ */
+const vasya = { name: "Вася", age: 23 };
+const masha = { name: "Маша", age: 18 };
+const vovochka = { name: "Вовочка", age: 6 };
+
+const people = [vasya, masha, vovochka];
+const clone = people.slice();
+// clone.sort((a, b) => a.age - b.age);
+clone.sort(function (a, b) {
+  return a.age - b.age;
+});
+console.log(clone);
+
+// теперь people: [vovochka, masha, vasya]
+console.log(clone[0].age); // 6
+
+console.log("");
+
+/**
+ * Добавить класс в строку
+ *
+ * В объекте есть свойство className,
+ * которое содержит список «классов» – слов, разделенных пробелом. Например, className: 'open menu'
+ *
+ * Создайте функцию addClass(obj, cls), которая добавляет в список класс cls,
+ * но только если его там еще нет. Функция не должна добавлять лишних пробелов.
+ *
+ */
+const obj = {
+  className: "open menu",
+};
+
+function addClass(obj, item) {
+  const a = obj.className.split(" ");
+  if (!a.includes(item)) {
+    obj.className = obj.className + " " + item;
+  }
+}
+
+addClass(obj, "new"); // obj.className='open menu new'
+addClass(obj, "open"); // без изменений (класс уже существует)
+addClass(obj, "me"); // obj.className='open menu new me'
+
+console.log(obj.className); // "open menu new me"
+console.log("");
+
+/**
+ * Напиши функцию pluck, которая берет массив объектов и
+ * возвращает массив значений определенного поля.
+ * Функция не должна изменять исходный массив.
+ */
+const characters = [
+  { name: "Михаил", age: 36 },
+  { name: "Фёдор", age: 40 },
+];
+
+function pluck(arr) {
+  return arr.map(function (el) {
+    return el.name;
+  });
+}
+
+console.log(pluck(characters)); // ['Михаил', 'Фёдор']
```

Comment:
Why did you change function signature? `pluck` should accept 2 params: array and field name